libs/core/services/window.service.ts

```diff
@@ -28,7 +28,7 @@ export class WindowService {
         return this._platformWindow.require;
     }
 
-    public alert(msg: any): Promise<any> {
+    public alert(msg: any): Promise<void> {
```

Comment:
if function has return statement, why are we changing any to void?

Reply:
`void` is provided because `resolve();` is not passing any data when Promise is getting resolved